serverless/admin_guide/serverless-ha.adoc

```diff
@@ -9,10 +9,19 @@ toc::[]
 High availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs.
 In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
 
-HA in {ServerlessProductName} is available through leader election, which is enabled by default after the Knative Serving control plane is installed.
+HA in {ServerlessProductName} is available through leader election, which is enabled by default after the Knative Serving or Eventing control plane is installed.
 
 When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required.
 These controller instances compete to use a shared resource, known as the leader election lock.
 The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
 
-include::modules/serverless-config-replicas.adoc[leveloffset=+1]
+== Configuring high availability replicas on {ServerlessProductName}
+
+High availability (HA) functionality is available by default on {ServerlessProductName} for the `autoscaler-hpa`, `controller`, `activator`, `kourier-control`, and `kourier-gateway` controllers. These components are configured with two replicas by default.
+
+You modify the number of replicas that are created per controller by changing the configuration of `KnativeServing.spec.highAvailability` in the KnativeServing custom resource definition (CRD) and `KnativeEventing.spec.highAvailability` spec in the KnativeEventing CRD.
```

Comment:
`KnativeServing.spec.highAvailability` and `KnativeEventing.spec.highAvailability` is a bit inaccurate.

it would probably be better to say

"... by changing the configuration of `spec.high-availability.replicas` in the KnativeServing custom resource (CR)"

(we're modifying a specific instance of the CRD, so it is a KnativeServing custom resource, not their definition. )

(same for KnativeEventing)